build/optaplanner-build-parent/pom.xml

```diff
@@ -874,6 +874,23 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
       </plugin>
+      <plugin>
```

Comment:
```suggestion
      <!-- Do not move to full profile. Otherwise upstream Kogito dependency changes can break our build without Kogito CI catching it. -->
      <plugin>
```